scripts/replace_parameter.sh

```diff
@@ -1,27 +1,69 @@
 #!/bin/bash
 
 usage() {
-	echo './replace_parameter.sh <REGISTER> <VALUE> <LINK>'
+
+    echo './replace_parameter.sh <REGISTER> <LINK> <VALUE>'
+	echo './replace_parameter.sh -f <FILENAME> <REGISTER> <LINK>'
 	echo ''
 	echo '	REGISTER - register to be updated dropping the "CFG_" substring'
 	echo ''
 	echo '	VALUE - value in decimal to be assigned to REGISTER'
+    echo ''
+    echo '	FILENAME - name of a file containing a list of vfat number/value pairs'
 	echo ''
-	echo '	Example:'
-	echo ''
-	echo '		./replace_parameter.sh PULSE_STRETCH 4 1'
+	echo '	Examples:'
 	echo ''
-
-	kill -INT $$; 
+	echo '		./replace_parameter.sh PULSE_STRETCH 1 4'
+	echo '		./replace_parameter.sh -f /path/to/NominalDacValues.txt PULSE_STRETCH 1'
```

Comment:
why did you remove the `kill -INT $$`?
it would make all the `exit`s you have redundant (and sometimes `exit` can have unintended consequences, referring to @bdorney's comment about testing good and bad usage)

Reply:
I removed that `kill -INT $$` because @bdorney requested an `echo` statement after the usage: https://github.com/cms-gem-daq-project/gemctp7user/pull/38#discussion_r229235451. Now that `echo` statement comes before the usage, so I just added the `kill -INT $$` back.

Reply:
My comment doesn't prevent you from using `kill -INT $$`.  The point is that the user should know what they did wrong.  The way you tell them this depends on the way they called the function.  Ideally you should:

1. provide the general use case,
2. if the script thinks they are trying to write all values to all registers and they screw up, provide that usage case,
3. if the script thinks they are providing a file to write unique values, provide that usage case.

This could be accomplished by having a general, and two specific `usage()` functions and still make use of `kill -INT $$`.

Reply:
OK, agreed. But I think the currently committed version with `kill -INT $$` in the `usage`  function and the `echo` statements on lines `45` and `53` resolves this issue, right?